bytecomp/bytegen.ml

```diff
@@ -769,6 +770,15 @@ let rec comp_expr stack_info env exp sz cont =
       let cont = add_pseudo_event loc !compunit_name cont in
       comp_args stack_info env args sz
         (Kmakeblock(List.length args, tag) :: cont)
+  | Lprim(Pmakelazyblock tag, [arg], loc) ->
+      let tag =
+        match tag with
+        | Lazy_tag -> Config.lazy_tag
+        | Forward_tag -> Obj.forward_tag
```

Comment:
Could this be a shared function somewhere? It's also repeated in `Cmmgen`

Reply:
Maybe `Lazy.tag_of_lazy_tag`?

Reply:
Done